kubernetes/samples/scripts/create-weblogic-domain/domain-home-on-pv/wlst/istio-fix-domain.py

```diff
@@ -0,0 +1,131 @@
+# Copyright 2018, 2019, Oracle Corporation and/or its affiliates.  All rights reserved.
+# Licensed under the Universal Permissive License v 1.0 as shown at http://oss.oracle.com/licenses/upl.
+
+def getEnvVar(var):
+  val=os.environ.get(var)
+  if val==None:
+    print "ERROR: Env var ",var, " not set."
+    sys.exit(1)
+  return val
+
+domain_uid                   = getEnvVar("DOMAIN_UID")
+server_port                  = int(getEnvVar("MANAGED_SERVER_PORT"))
+domain_path                  = getEnvVar("DOMAIN_HOME")
+cluster_name                 = getEnvVar("CLUSTER_NAME")
+admin_server_name            = getEnvVar("ADMIN_SERVER_NAME")
+admin_server_name_svc        = getEnvVar("ADMIN_SERVER_NAME_SVC")
+admin_port                   = int(getEnvVar("ADMIN_PORT"))
+domain_name                  = getEnvVar("DOMAIN_NAME")
+t3_channel_port              = int(getEnvVar("T3_CHANNEL_PORT"))
+t3_public_address            = getEnvVar("T3_PUBLIC_ADDRESS")
+number_of_ms                 = int(getEnvVar("CONFIGURED_MANAGED_SERVER_COUNT"))
+cluster_type                 = getEnvVar("CLUSTER_TYPE")
+managed_server_name_base     = getEnvVar("MANAGED_SERVER_NAME_BASE")
+managed_server_name_base_svc = getEnvVar("MANAGED_SERVER_NAME_BASE_SVC")
+domain_logs                  = getEnvVar("DOMAIN_LOGS_DIR")
+script_dir                   = getEnvVar("CREATE_DOMAIN_SCRIPT_DIR")
+
+# Update Domain for Istio
+readDomain(domain_path)
+
+cd('/Servers/' + admin_server_name)
+create('istio-probe', 'NetworkAccessPoint')
+cd('/Servers/%s/NetworkAccessPoints/istio-probe' % admin_server_name)
+set('Protocol', 'http')
+set('ListenAddress', '127.0.0.1')
+set('PublicAddress', domain_uid + '-' + admin_server_name)
+set('ListenPort', 8888)
+set('HttpEnabledForThisProtocol', true)
+set('TunnelingEnabled', false)
+set('OutboundEnabled', false)
+set('Enabled', true)
+set('TwoWaySslEnabled', false)
+set('ClientCertificateEnforced', false)
```

Comment:
If istio always needs to add a special port for the ready probe, the introspector should add it via custom sit-config, and the port-number should be configurable so that end-users don't need to make any changes to their config to handle the situation.  

Also, even though the listen address is 127.0.0.1, which I suppose 'hides' the port from resources external to the pod?, I wonder if it should be locked down further and made SSL instead of plain-text.